src/cleanlab_codex/internal/validator.py

```diff
@@ -21,6 +24,17 @@
     "context_sufficiency": "is_not_context_sufficient",
 }
 
+REWRITE_QUERY = """Given a conversational Message History and a Query, rewrite the Query to be a self-contained question. If the query is already self contained, return it as-is.
+
+Query: {query}
+
+--
+Message History: \n{messages}
```

Comment:
You're already using triple-quotes.

```suggestion
Message History:
{messages}
```